pkg/endpoints/cluster.go

```diff
@@ -136,3 +139,83 @@ func (r Resource) GetProperties(request *restful.Request, response *restful.Resp
 	properties := Properties{InstallNamespace: os.Getenv("INSTALLED_NAMESPACE")}
 	response.WriteEntity(properties)
 }
+
+// Get dashboard version
+func (r Resource) GetDashboardVersion(request *restful.Request, response *restful.Response) {
```

Comment:
Would it make sense to add these to GetProperties? That way we could get both versions (and other future details) in 1 request